doc/Core.xml

```diff
@@ -2169,43 +2183,65 @@ DATE: when response was generated
+      	<para>Both digest authentication and WS-Security provide only a rudimentary level of security. In a system where security is important, it is recommended to always configure the device for TLS-based access (see Security Service). Digest authentication or WS-Security messages can be combined with TLS security for a client and server authentication. A protected transport level of security will provide an acceptable level of security in many systems.</para>
+        <para>Bearer tokens in HTTP headers do not provide security. It is recommended to always configure the device for TLS-based access (see Security Service). A protected transport level of security will provide an acceptable level of security in many systems. Bearer tokens should never be used without TLS.</para>
         <para>An ONVIF compliant device should authenticate an RTSP request at the RTSP level. If HTTP is used to tunnel the RTSP request the device shall not authenticate on the HTTP level.</para>
-        <para>When authenticating RTSP or HTTP methods, an ONVIF compliant device shall use digest authentication [RFC 2617]. The credentials shall be managed with the GetUsers, CreateUsers, DeleteUsers and SetUser methods. If the device also supports WS-Security, the same set of credentials shall be used.</para>
+        <para>When authenticating RTSP or HTTP methods, an ONVIF compliant device shall use digest authentication [RFC 2617] or JWT-based authorization. The credentials shall be managed with the GetUsers, CreateUsers, DeleteUsers and SetUser methods. If the device also supports WS-Security, the same set of credentials shall be used.</para>
+      </section>
+      <section>
+        <title>Authentication over SCTP</title>
+        <para>The services defined in this standard, whenever consumed overt SCTP, shall be protected using digest authentication according to [WS-BinarySecurityToken].</para>
+        <para>Since HTTP headers are not present when consuming the services over SCTP, upon authentication failure the device shall return only the a SOAP:Fault env:Sender ter:NotAuthorized error on the WS level.</para>
+        <para><xref linkend="AuthenticationFlowSCTP" /> summarizes the authentication of a web service request by a server over SCTP.</para>
+        <figure xml:id="AuthenticationFlowSCTP">
+          <title>Authentication of a WS request by a server over SCTP</title>
+          <mediaobject>
+            <imageobject>
+              <imagedata fileref="media/Core/image7.svg" contentwidth="150mm" />
+            </imageobject>
+          </mediaobject>
+        </figure>
+        <para>WS-Security BinarySecurityToken does not provide security. It is recommended to always configure the device for TLS-based access (see Advanced Security Service). A protected transport level of security will provide an acceptable level of security in many systems. WS-Security BinarySecurityToken should never be used without TLS.</para>
```

Comment:
"Bearer tokens should never be used without TLS", should (shall?) a device block a request or is it ok to accept the request? As the sentence is now it doesn't state what the device should do.